### ISSUE TITLE
[1.8.9]Request: Disable Villager

### ISSUE THREAD

**Issue by iPencil** [BluSunrize/ImmersiveEngineering#1185]
Could we get a config option to disable the IE villager? 
It's not really all that useful and it currently causes a crash with Spongeforge everytime the villager is loaded:
https://github.com/SpongePowered/SpongeForge/issues/676

I can understand that you might not want to support Sponge, but without a proper protection mod/plugin most servers cannot function.

Consider adding an option to simply disable this villager, unless you can/want to fix the issue (if it's even your fault :P )

**Comment by BluSunrize** [comment]
Ah this is actually well spotted.
You see the IE engineer currently isn't implemented. 18.9 gives me nothing to work with, villagers are completely broken, so I actually didn't add the villagers back. However, the house still tries to spawn them, despite the profession never being registered. The result of that is the given crash that only sponge seems to highlight.

tl;dr: Legitimate issue, will fix in the next release.

**Comment by iPencil** [comment]
Ah, I wasn't aware that 1.8.9 fucked up villagers.

Well good to know! :D Thanks for the quick response, I'll be waiting for that update then :)
